.gitignore

```diff
@@ -61,4 +61,4 @@ bld/
 *.VC.opendb
 
 # Visual C++ Solution
-*.sln
+*.VC.db

```

Comment:
I pushed the final fix (for now) that makes guard pages work on x64 (plus
the code is simplified on 32).

I will be using this lib to demonstrate x64dbg on a conference so thanks
for your work 👍
On Fri, 1 Jul 2016 at 11:44, Noteworthy <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/LordNoteworthy/al-khaser/pull/50#discussion_r69274799>
> :
>
> > @@ -61,4 +61,4 @@ bld/
> >  *.VC.opendb
> >
> >  # Visual C++ Solution
> > -*.sln
> > +*.VC.db
>
> Definitely, the*.VC.db have to be ignored and good that we are not
> ignoring now the solution file.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LordNoteworthy/al-khaser/pull/50/files/31e98461c809b1cab96b02cf43c52a5eb9cd006e#r69274799>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACWCmXADKwwaPbjAhvkM93rjdUT3wIpCks5qROFugaJpZM4I_rAC>
> .
>